src/scss/grommet-core/_objects.layer.scss

```diff
@@ -111,6 +111,14 @@
       max-height: calc(100vh - #{double($inuit-base-spacing-unit)});
       max-width: calc(100vw - #{double($inuit-base-spacing-unit)});
       transform: translate(50%, 50%);
+
```

Comment:
Yeah I just validated it, remove these lines fix the issue. Do you mind to update your pull request to reflect it? Please remove from line 60 to 71.